cfgov/v1/models/learn_page.py

```diff
@@ -339,15 +339,15 @@ class EventPage(AbstractFilterPage):
         help_text='Check if this event will be streamed live. This causes the '
                   'event page to show the parts necessary for live streaming.'
     )
-    live_stream_url = models.URLField(
-        "URL",
+    live_video_id = models.CharField(
```

Comment:
Unfortunately we'd only be able to consolidate the two uses in EventPage, since the use in VideoPlayer is of a different type (a Wagtail StreamField block type vs. a Django field type). Since these two are right near each other, I'm inclined to let it be until we need it [in a third place](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)).